javascript/react/src/hooks/uploads.ts

```diff
@@ -106,10 +106,10 @@ export function useHandleZipUpload(
 		},
 		[csvHandler, jsonHandler, onUpdateTables],
 	)
-	return useHandleOnUploadClick(['.zip'], handler)
+	return useUploadFileHandler(['.zip'], handler)
 }
 
-export function useHandleOnUploadClick(
+export function useUploadFileHandler(
```

Comment:
Let's rename this `useHandleFileUpload` to keep the naming convention used in `useHandleCsvUpload` and `useHandleJsonUpload`